torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -519,7 +525,8 @@ void RequestCallbackNoPython::processRunWithProfilingReq(
         rpcWithProfilingReq.wrappedRpc(),
         wrappedMsgType,
         messageId,
-        wrappedRpcResponseFuture);
+        wrappedRpcResponseFuture,
+        {});
```

Comment:
this means this PR won't fix the cuda stream sync problem when the profiler is enabled? This is totally fine, we can add that in a followup PR, but let's don't forget to add it before the branch cut. 

cc @rohan-varma

Reply:
@pbelevich we can create an issue to track this so that we won't forget.